test/Nancy.Tests/project.json

```diff
@@ -2,8 +2,8 @@
     "version": "1.4.1.0",
     "dependencies": {
         "dotnet-test-xunit": "2.2.0-preview2-build1029",
-        "FakeItEasy": "3.0.0-rc001",
-        "FakeItEasy.Analyzer": "3.0.0-rc001",
+        "FakeItEasy": "3.0.0",
+        "FakeItEasy.Analyzer": "3.0.0",

```

Comment:
Oh, @thecodejunkie, here's a fun example that comes to mind for no particular reason.

```
Nancy/test/Nancy.ViewEngines.Razor.Tests/TextResourceFinderFixture.cs(46,46): warning FakeItEasy0002: Non virtual member '_' cannot be intercepted.
```